Neo4j.Driver/Neo4j.Driver/Internal/Helpers/TemporalHelpers.cs

```diff
@@ -63,6 +62,19 @@ public static long ToNanoOfDay(this IHasTimeComponents time)
                    time.Nanosecond;
         }
 
+#if NET6_0_OR_GREATER
+        public static long ToNanoOfDay(this TimeOnly time)
+        {
+            return (time.Hour * NanosPerHour) + (time.Minute * NanosPerMinute) + (time.Second * NanosPerSecond) +
+                   time.Millisecond * NanosPerMillisecond;
```

Comment:
Very minor nit pick. Add brackets to the last pair for consistency.  Obviously makes no difference to the calculation.